### DISCUSSION TITLE
NEAR Docs Translation: https://crwd.in/near-docs

### DISCUSSION THREAD

**Discussion by think-in-universe** [near-x/learn-near-community#43] [General]
## Introduction

Welcome to NEAR Docs (https://docs.near.org/) translation project. 

### Roles

1. Translator: translate docs
2. Proof Reader: help proof reading of docs
3. Manager: help manage membership

### How to Become a Translator

To become a certified NEAR docs translator, you need to pass a pre-translation test by working on a selected document translation. A proof reader will review your document, and if your translation proves to be high-quality, your application will be accepted and you'll be accepted as a NEAR Certified Translator. 

### How to Translate

Before you start working on the translation of an article, you need to get pre-approval from the community first. The reason is to make sure the plans are transparent in the community, and the proof readers can help to review and approve the translation once it's ready. 

Once your plan is pre-approved by the community, you can just go to the Crowdin project: https://crwd.in/near-docs and work on the translation of the specific markdown file. 

### Translation Plan

For each language, the current plan is to translate all the documents with the following priorities.

**Website**

1. website (except docusaurus-plugin-content-blog)

**Developer**

1. concepts
2. develop
3. tools
4. api
5. tutorials
6. videos

**Community**

1. community

**Validator & Integration**

1. validator
2. tokens
3. roles
4. faq

![image](https://user-images.githubusercontent.com/46699230/138542556-0d6f0582-cb9a-4031-8610-e7f82f342713.png)

### Proof Reading and Go-To Production

For each document, we need no less than two proof readers to review the translations. The translator or proof reader can propose different translations if he/she doesn't agree the existing ones, the translations that receive the most votes will be accepted and displayed in the production version.

**Comment by JuEnPeHa** [comment]
Hello Sir, my name is Juan, I'd like to help with the translations to Spanish, thank you.

At your service.
